src/main/java/com/truelayer/java/payments/entities/paymentdetail/RetryOption.java

```diff
@@ -0,0 +1,14 @@
+package com.truelayer.java.payments.entities.paymentdetail;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+@Getter
+public enum RetryOption {
+    RESTART("restart");
```

Comment:
This is currently the only option available (confirmed by @tl-Roberto-Mancinelli )